modyn/supervisor/internal/pipeline_executor/evaluation_executor.py

```diff
@@ -134,17 +129,14 @@ def run_pipeline_evaluations(
             trigger_id: The trigger id to evaluate.
             training_id: The training id to evaluate.
             model_id: The model id to evaluate.
-            first_timestamp: Start of the interval to evaluate.
-            last_timestamp: End of the interval to evaluate.
+            first_timestamp: Start of the training interval.
```

Comment:
I have just started to scroll through the PR but given the confusion I would like to clarify this change. These intervals have nothing to do with training, right @robinholzi ? They describe how the eval dataset is sliced. Hence I am not sure if there is a misunderstanding here

Reply:
In the Slicing strategy e.g. these intervals are completely independent from training intervals (unless the triggers are configured to follow the same schedule ;) ). I think we should keep the old description.

Reply:
but why within the method we pass them to an arg of `training_interval`?
https://github.com/eth-easl/modyn/blob/60d3baa7523d21197dec85262fb8c31662d656c2/modyn/supervisor/internal/pipeline_executor/evaluation_executor.py#L159

Reply:
Yes my question is we pass these two parameters to the `training_interval` of this function, so their description in the docstring should be "training interval" right? Or I mistook anything?

Reply:
Reading the code again, this seems indeed to be the interval of training for this model, so your updated description might be correct. @robinholzi WDYT?